data/pistol/cip.json

```diff
@@ -1,96 +1,1090 @@
 {
-    "5.45 x 18": {},
-    "5.75 Velodog": {},
-    "6.35 Browning": {},
-    "7 x 49 Guido J. Wasser": {},
-    "7 Penna": {},
-    "7mm Penna L": {},
-    "7.5 FK": {},
-    "7.5 Ordonnance Suisse": {},
-    "7.62 x 25 Tokarev": {},
-    "7.62 Nagant": {},
-    "7.63 Mauser": {},
-    "7.65 Browning": {},
-    "7.65 Long": {},
-    "7.65 Parabellum": {},
-    "8mm Gasser": {},
-    "8mm Lebel": {},
-    "8mm Steyr": {},
-    "9 x 18": {},
-    "9 x 20 VGW": {},
-    "9 x 21": {},
-    "9 x 22 Major": {},
-    "9 x 25 Super Automatic G": {},
-    "9mm Makarov": {},
-    "9mm Browning court": {},
-    "9mm Browning long": {},
-    "9mm Fast Accurate Reliable": {},
-    "9mm FX & CQT": {},
-    "9mm Luger": {},
-    "9mm Mauser": {},
-    "9mm Steyr": {},
-    "10 x 22 T": {},
-    "10mm Automatic": {},
-    "10mm Fast Accurate Reliable": {},
-    "10.4 Ordinanza Italiana": {},
-    "11mm 73": {},
-    "22 PICRA": {},
-    "22 Remington Jet Magnum": {},
-    "221 Remington Fireball": {},
-    "260 PICRA": {},
-    "30 PICRA": {},
-    "30-357 Armi e Tiro": {},
-    "32 Harrington & Richardson Magnum": {},
-    "32 Long Colt": {},
-    "32 Short Colt": {},
-    "32 Smith & Wesson": {},
-    "32 Smith & Wesson Long": {},
-    "32 Smith & Wesson Long Wad Cutter": {},
-    "320 Long": {},
-    "320 Short": {},
-    "357 Automatic Magnum": {},
-    "357 Magnum": {},
-    "357 Magnum (carb)": {},
-    "357 Maximum": {},
-    "357 SIG": {},
-    "38 Long Colt": {},
-    "38 Short Colt": {},
-    "38 Smith & Wesson. Colt New Police": {},
-    "38 Special": {},
-    "38 Special (carb)": {},
-    "38 Special Army Marksmanship Unit": {},
-    "38 Special Wad Cutter": {},
-    "38 Super Automatic": {},
-    "38/357 FX": {},
-    "38-45 Automatic Colt Pistol": {},
-    "380 Long": {},
-    "380 Short": {},
-    "40 Smith & Wesson": {},
-    "41 Action Express": {},
-    "41 Long Colt": {},
-    "41 Remington Magnum": {},
-    "44 Colt": {},
-    "44 Remington Magnum": {},
-    "44 Remington Magnum (carb)": {},
-    "44 Smith & Wesson Russian": {},
-    "44 Smith & Wesson Special": {},
-    "44 Smith & Wesson Special (carb)": {},
-    "45 Automatic": {},
-    "45 Automatic Rim": {},
-    "45 Colt": {},
-    "45 Colt (carb)": {},
-    "45 Glock Auto Pistol": {},
-    "45 HP": {},
-    "45 Smith & Wesson Schofield": {},
-    "45 Smith & Wesson Schofield (carb)": {},
-    "45 Winchester Magnum": {},
-    "450 SCH": {},
-    "450 Short": {},
-    "454 Casull": {},
-    "455 MK II": {},
-    "460 Smith & Wesson Magnum": {},
-    "475 Linebaugh": {},
-    "480 Ruger": {},
-    "50 Action Express": {},
-    "500 Smith & Wesson Magnum": {}
-}
+  "10 x 22 T" : {
+    "name" : "10 x 22 T",
+    "names" : [ ],
+    "specs" : {
+      "coal_mm" : 22.7,
+      "coal_inch" : 0.8937012699999999
```

Comment:
Did you convert the MM value to obtain the INCH value?

Reply:
Yes exactly, had hoped that i wouldn't run here into that presentation problem with double.
Had extra checked it for multiple cartridges and there it looked okay.

Best thing would be to remove it i think, or do you have any ideas how we can work around this issue?

Reply:
It's fine to convert it, but there are some gotchas. Doing so directly in a programming lang can cause issues because floating point numbers don't compute like a calculator.  For example, this is what a calculator will provide:

    22.7 / 25.4 = 0.8937007874

Often, there will be a library available for unit conversion. (Python: [pint](https://pint.readthedocs.io/en/0.9/), JS: [Math.js](https://mathjs.org/docs/datatypes/units.html))

I recommend using something like that, then spot-checking as many as you can where the result is predictable. I'm not exactly sure what level of precision it should be rounded too -- maybe compare with the SAAMI data?